plans/extend_ca_cert.pp

```diff
@@ -40,6 +40,14 @@
     }
   }
 
+  if $is_pe {
+    $crl_results = run_task('ca_extend::check_crl_cert', $targets, '_catch_errors' => true).first
+    unless $crl_results.ok {
```

Comment:
Related to my comment in the task, this would need to check the status of the crl from the return value of the task.  The task could return failure for a number of reasons, but we'd only want to create a new CRL if the task run successfully and returned a result with a `crl_status` of `expired`.  We can check that value with something like:

```
if $crl_results.value['crl_status'] == 'expired' {
```